portfolio.html

```diff
@@ -13,31 +13,37 @@
 	<body>		
 		<nav class="nav-bar">
 			<a class="navigation-links" href="#home">home</a> 
-<!--need to root home button to different part of page-->
 			<a class="navigation-links" href="#about-me">about me</a>
 			<button class="cv-button">download my CV!</button>
 			<a class="navigation-links" href="#portfolio">portfolio</a>
 			<a class="navigation-links" href="#contact-me">contact me</a>	
 		</nav>	
 	
-		<section>
-			<div class="welcome-banner">
-				<h1 class="hi-there-welcome-text">
+		<section class="welcome-banner">
+			<div>
+				<h1 class="small-title">
+					<img class="sparkle-inline" src="sparkles.gif" alt="sparkle">
 					<a id="home">hi there</a>
 				</h1>
-				<h2 class="welcome-text">
+			</div>
+			<div>
+				<h2 class="large-title">
 					welcome to my portfolio!
 				</h2>
-			
 			</div>
-		
-			<div class="scallop-mid-section">
+			<div>
+		</section>
+		<section class="about-section">
+			<div>
 				<h2>
 					<a class="about-me-title" id="about-me">About me</a>
 				</h2>
-				<p class="about-me-text">Hello, I'm Charlotte - a software developer supreme in the making! I've previously worked within the Events Industry so making things come together for clients is in my nature.</p>
-				<img class="profile-photo" src="Charlotte-Buckley-2021.jpeg" alt="photo of Charlotte Buckley">
+				<p class="about-me-text">Hello, I'm Charlotte - a software developer supreme in the making! I've previously worked within the Events Industry so delivering projects for clients is in my nature.</p>
```

Comment:
have you considered using a line break rather than two p's? Would save on double class naming